pkg/helpers/helpers_test.go

```diff
@@ -73,7 +73,7 @@ func TestPointerToBool(t *testing.T) {
 		t.Fatalf("expected PointerToBool(true) to return *true, instead returned %#v", ret)
```

Comment:
nit: these print statements should print %d, boolVar instead of true/false strings

Reply:
Sure